src/mlmc/quantity.py

```diff
@@ -1,30 +1,105 @@
-import abc
 import numpy as np
 import copy
 import operator
 from inspect import signature
 from memoization import cached
 from scipy import interpolate
-from typing import List, Tuple
+from typing import List
 from mlmc.sample_storage import SampleStorage
-from mlmc.sim.simulation import QuantitySpec
+from mlmc.quantity_spec import QuantitySpec
+from mlmc.quantity_types import QType, ScalarType, BoolType, ArrayType, DictType, TimeSeriesType, FieldType
 
-CHUNK_SIZE = 512000  # bytes in decimal
+
+def _get_quantity_info(args_quantities, get_quantity_storage=False):
```

Comment:
One commit includes:
- split of quantity.py
- "fix tests" 
- substantial changes in the quantity, new code
Too much changes. In one commit and together with other changes in the PR.